client/packages/service-message/components/MarkdownViewer.tsx

```diff
@@ -0,0 +1,25 @@
+import { marked } from 'marked';
+import dompurify from 'dompurify';
+import { FC } from 'react';
+import { Typography } from '@equinor/eds-core-react';
+import styled from 'styled-components';
+
+type MarkdownViewerProps = {
+    markdown: string;
+};
+
+const StyledTypography = styled(Typography)`
+    overflow: auto;
+`;
+
+
+
+const MarkdownViewer: FC<MarkdownViewerProps> = ({ markdown }) => {
+    return (
+        < StyledTypography variant="body_long"
+            dangerouslySetInnerHTML={{ __html: dompurify.sanitize(marked.parse(markdown)) }}
```

Comment:
Are you sure there is not a better way to do this? dangerouslySetInnerHtml

Reply:
This one is taken from Fusion API repo im not sure, but i think one need it for the markdown to properly show. 
we could make an issue on it an research it furder.